tests/requirements.txt

```diff
@@ -1,6 +1,5 @@
 # Tests requirements
 attrs
-dataclasses; python_version == '3.6'
 pyftpdlib
 pytest
 pytest-cov==3.0.0
```

Comment:
Se also the `uvloop` lines below, the first one can be removed and the second one simplified.